2.6/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -151,8 +147,10 @@ function mongo_primary_member_addr() {
 
 # mongo_remove removes the current MongoDB from the cluster
 function mongo_remove() {
-  local primary_addr
-  primary_addr="$(mongo_primary_member_addr)"
+  local primary_addr="$(mongo_primary_member_addr ||  true)"
```

Comment:
The comment is not to say "ignore result of the command", but why are we doing it.

Why are we doing it? Why is `|| true` necessary?